### PR TITLE
Added Cla

### PR DESCRIPTION
**Pull request by atiqsayyed**
Added cla

### CODE REVIEW

File: agreements/atiqsayyed.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### 8.
+You agree to notify the `Company` of any facts or circumstances of which you become aware that would make these
+representations inaccurate in any respect.
+
+
+### Please sign below
+
+Full-Name              : Atiq Sayyed
+Date                   : 010/27/2015
```
Comment (hrj) [inline agreements/atiqsayyed.md]: Thanks for the PR. Please check the date. Would be best in this format : `23 Sept 2015`

**Comment by atiqsayyed** [comment]
Fixed the Date of birth

File: agreements/atiqsayyed.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### 8.
+You agree to notify the `Company` of any facts or circumstances of which you become aware that would make these
+representations inaccurate in any respect.
+
+
+### Please sign below
+
+Full-Name              : Atiq Sayyed
+Date                   : 27 Oct 1990
```
Comment (hrj) [inline agreements/atiqsayyed.md]: Atiq, it's not date of birth field!

Just today's date is sufficient :smile:

**Comment by atiqsayyed** [comment]
Oh, m really sorry. . 
This is what happens when you come to office after 5 days and flooded with work. . .  .

fixed it. .

**Comment by hrj** [comment]
Thanks for the CLA and fixes, despite the flood!

**Pull request merged**